code/modules/mining/machine_redemption.dm

```diff
@@ -88,13 +93,6 @@
 					else
 						process_sheet(O)
 						i++
-		var/msg = "Now available in the Cargo Bay: \n"
```

Comment:
Just add
    
    if(i > 0)

that way the message is only sent if there were any new ores processed this tick, which is probalby what was intended.